### ISSUE TITLE
`-l` with more than one language results in an error

### ISSUE THREAD

**Issue by strobi** [jbarlow83/OCRmyPDF#578]
**Describe the bug**
Calling ``ocrmypdf`` with more than one language (e.g. ``-l deu+eng``) results in an error. It works correctly if only one language is specified. The corresponding tesseract languages are correctly installed and worked before the update.

**To Reproduce**
```bash
ocrmypdf -v1 -l deu+eng input.pdf output.pdf

ocrmypdf 10.0.0
Running: ['tesseract', '--list-langs']
Running: ['tesseract', '--version']
Running: ['tesseract', '--list-langs']
Traceback (most recent call last):
  File "/usr/local/bin/ocrmypdf", line 8, in <module>
    sys.exit(run())
  File "/usr/local/Cellar/ocrmypdf/10.0.0/libexec/lib/python3.8/site-packages/ocrmypdf/__main__.py", line 54, in run
    check_options(options, plugin_manager)
  File "/usr/local/Cellar/ocrmypdf/10.0.0/libexec/lib/python3.8/site-packages/ocrmypdf/_validation.py", line 256, in check_options
    check_options_languages(options, plugin_manager)
  File "/usr/local/Cellar/ocrmypdf/10.0.0/libexec/lib/python3.8/site-packages/ocrmypdf/_validation.py", line 84, in check_options_languages
    msg += lang + '\n'
TypeError: unsupported operand type(s) for +: 'generator' and 'str'
```

**Example file**
As far as I can tell, the error occurs with all files, including files from ``tests/resources``.

**Expected behavior**
OCRmyPDF should accept and recognize more than one language and pass them to tesseract.

**System**
 - OS: macOS 10.15.5
 - OCRmyPDF Version: 10.0.0
 - How did you install ocrmypdf? Homebrew

**Comment by jbarlow83** [comment]
Yep, that's a regression. 

Fortunately the alternate syntax still works: `-l deu -l eng`.

**Comment by rbrito** [comment]
Thanks a lot for the workaround, @jbarlow83.

**Comment by jbarlow83** [comment]
Fixed in v10.0.1
